captum/optim/_core/loss.py

```diff
@@ -371,3 +573,7 @@ def __call__(self, targets_to_values: ModuleOutputMapping) -> torch.Tensor:
         else:
             activations = activations * self.weights.view(1, -1, 1, 1)
         return activations
+
+
+def default_loss_summarize(loss_value):

```

Comment:
Suggestion: We should include type hints for the loss summarizer.

Reply:
Done!